MARIE_SEQ2SEQ/README.md

```diff
@@ -0,0 +1,11 @@
+# Marie4
+
```

Comment:
This documentation is pretty minimalistic, mildly speaking. It would be massively helpful to add a few sentences on what Marie is, what it does, what principles it is based on, what technologies it uses (e.g. what is Triton and what is it used for), etc.

Reply:
Here I've added a short description of Marie is. On the application architecture, concerning what Triton is and what it is used for, I've updated `deployment/README.md` to reflect this info.